Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponseEventCollector/ProofpointThreatResponseEventCollector.yml

```diff
@@ -0,0 +1,153 @@
+category: Email Gateway
+commonfields:
+  id: ProofpointThreatResponseEventCollector
+  version: -1
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  required: true
+  type: 0
+- display: ""
+  displaypassword: API Key for the authentication.
+  name: credentials
+  type: 9
+  required: true
+  hiddenusername: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- defaultvalue: 'false'
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- additionalinfo: The time range for the initial data fetch. If timeout errors occur, consider changing this value.
+  defaultvalue: 3 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  name: first_fetch
+  type: 0
+  required: false
+- defaultvalue: '100'
+  display: Fetch limit - maximum number of incidents per fetch
+  name: fetch_limit
+  required: false
+  type: 0
+- defaultvalue: '6 hours'
+  additionalinfo: The time range between create_after and created_before that is sent to the API when fetching
+    older incidents. If timeout errors occur, consider changing this value.
+  display: Fetch delta - The delta time in each batch. e.g. 1 hour, 3 minutes.
+  name: fetch_delta
+  required: false
+  type: 0
+- display: Fetch incidents with specific event sources. Can be a list of comma separated values.
+  name: event_sources
+  required: false
+  type: 0
+- display: Fetch incidents with specific 'Abuse Disposition' values. Can be a list of comma separated values.
+  name: abuse_disposition
+  required: false
+  type: 0
+- display: Fetch incident with specific states.
+  name: states
+  options:
+  - new
+  - open
+  - assigned
+  - closed
+  - ignored
+  required: false
+  type: 16
+- additionalinfo: You can find this value by navigating to Sources -> JSON event source -> POST URL.
+  display: POST URL of the JSON alert source.
+  hidden: false
+  name: post_url_id
+  required: false
+  type: 0
+description: Use the Proofpoint Threat Response integration to orchestrate and automate incident response.
```

Comment:
please verify this description with a TPM

Reply:
Meital is out of the office.  This is the description of the integration that fetches incidents.